index.html

```diff
@@ -0,0 +1,165 @@
+            </section>
+            <p class="intro-date">2022.6.2(MON)~3(TUE)</p>
+            <span class="intro-location">In Amsterdam</span>
+        </div>
+    </section>
+    <!-- Main program -->
+    <section class="program">
+        <div class="program-title">
+            <h1>Main Program</h1>
+            <span class="program-line"></span>
+        </div>
+        <div class="program-card">
+            <div class="card">
+                <section class="card-title">
+                    <i class="fa fa-diamond" aria-hidden="true"></i>
+                    <h2>Card heading</h2>
+                </section>
+                <span>
+                    Apply for our 2022 Metting and Let's meet in Amsterdam
+                </span>
+            </div>
+            <div class="card">
+                <section class="card-title">
+                    <i class="fa fa-diamond" aria-hidden="true"></i>
+                    <h2>Card heading</h2>
+                </section>
+                <span>
+                    Apply for our 2022 Metting and Let's meet in Amsterdam
+                </span>
+            </div>
+            <div class="card">
+                <section class="card-title">
+                    <i class="fa fa-diamond" aria-hidden="true"></i>
+                    <h2>Card heading</h2>
+                </section>
+                <span>
+                    Apply for our 2022 Metting and Let's meet in Amsterdam
+                </span>
+            </div>
+            <div class="card">
+                <section class="card-title">
+                    <i class="fa fa-diamond" aria-hidden="true"></i>
+                    <h2>Card heading</h2>
+                </section>
+                <span>
+                    Apply for our 2022 Metting and Let's meet in Amsterdam
+                </span>
+            </div>
+            <div class="card">
+                <section class="card-title">
+                    <i class="fa fa-diamond" aria-hidden="true"></i>
+                    <h2>Card heading</h2>
+                </section>
+                <span>
+                    Apply for our 2022 Metting and Let's meet in Amsterdam
+                </span>
+            </div>    
+    </div>
+    <div class="program-apply">
+        <button>Join us in Amsterdam</button>
+    </div>
+    </section>
+
+    <section class="speaker">
+        <div class="speaker-title">
+            <h1>Feature Speaker</h1>
+            <span class="speaker-line"></span>
+        </div>
+    
+        <div class="speaker-class" id="speaker-id"></div>
+
+
+    </section>
+    <section class="partner">
+        <div class="partner-title">
+            <h1>Partner</h1>
+            <span class="partner-line"></span>
+        </div>
+        <div class="partner-company">
+            <div class="company">
+                <h5>GOOGLE</h5>
+            </div>
+            <div class="company">
+                <h5>mozilla</h5>
+            </div>
```

Comment:
- [ ] Please notice that the headline in this section should have a light color as in the reference design. 
- [ ] Also the orange border under the headline should be filled with the orange color and not outlined

|Your Design | Reference Design |
--|--
|![image](https://user-images.githubusercontent.com/18730106/152450622-21c41b0e-1030-4386-b469-9a9916403b38.png)|![image](https://user-images.githubusercontent.com/18730106/152450652-a5f937a7-4a3c-4d13-b3cb-c31ca04c9f5b.png)|